soc/arm/silabs_exx32/Kconfig

```diff
@@ -140,6 +140,11 @@ config CRYPTO_ACC_GECKO_TRNG
 	  Enable Entropy driver based on the CRYPTO_ACC module for Silicon Labs
 	  Gecko chips.
 
+config SOC_GECKO_SEMAILBOX
```

Comment:
what is using this? If nothing remove for now.

Reply:
Good catch. This symbol isn't used right now. I've removed it for now, and will reintroduce it in another PR.